pkg/cmd/server/server.go

```diff
@@ -728,10 +712,9 @@ func (s *server) runControllers(defaultVolumeSnapshotLocations map[string]string
 	}
 
 	enabledControllers := map[string]func() controllerRunInfo{
-		controller.BackupSync:        backupSyncControllerRunInfo,
-		controller.Backup:            backupControllerRunInfo,
-		controller.GarbageCollection: gcControllerRunInfo,
```

Comment:
After removing GC Controller from enabledControllers, it should be added into enabledRuntimeControllers, because Velero has a feature to disable specific controllers according to command line parameter.
If we register the controller by default, the disble controller feature will not work.
Looks like there is alreay refactored controllers not obeying this rule, I will try to fix that in backup-sync refactor PR.

Reply:
Yes, you're right, thanks for pointing it out.